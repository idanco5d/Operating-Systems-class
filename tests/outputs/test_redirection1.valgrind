==7442== Memcheck, a memory error detector
==7442== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==7442== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==7442== Command: /mnt/c/Users/illus/os/hw/1/smash
==7442== Parent PID: 7437
==7442== 
==7442== 
==7442== FILE DESCRIPTORS: 4 open (3 std) at exit.
==7442== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_redirection1.valgrind
==7442==    <inherited from parent>
==7442== 
==7442== 
==7442== HEAP SUMMARY:
==7442==     in use at exit: 0 bytes in 0 blocks
==7442==   total heap usage: 58 allocs, 58 frees, 80,623 bytes allocated
==7442== 
==7442== All heap blocks were freed -- no leaks are possible
==7442== 
==7442== For lists of detected and suppressed errors, rerun with: -s
==7442== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
