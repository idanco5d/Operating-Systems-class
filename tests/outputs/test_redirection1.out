smash> smash> smash pid is 9645
smash> smash: sending SIGKILL signal to 0 jobs:
